src/daemon/delayed_shutdown_timer.cpp

```diff
@@ -23,20 +23,38 @@
 namespace mp = multipass;
 namespace mpl = multipass::logging;
 
-mp::DelayedShutdownTimer::DelayedShutdownTimer(VirtualMachine* virtual_machine) : virtual_machine{virtual_machine}
+namespace
+{
+std::string shutdown_message(const std::chrono::minutes& time_left)
```

Comment:
I wouldn't expect the returned value to include the `wall `.

Reply:
Ok, but I did this to keep from having to add `wall` to every session.exec() call we make.  Would renaming the function make it more clear?  Maybe something like shutdown_message_command() or something?

Reply:
Or even pass the ssh session in here?

Reply:
Yeah, that works too:)

Reply:
Done